### DISCUSSION TITLE
Hide Whitespace option for commit details page

### DISCUSSION THREAD

**Discussion by ras-relativity** [community/community#27343] [Other Feature Feedback, Questions, & Ideas] [answered]
Add the 'Hide Whitespace' setting on Pull Requests to the commit details page.

Example of a PR with 'hide whitespace' available and on, versus the merged commit:
PR: https://github.com/microsoft/vscode/pull/155224/files?diff=unified&w=1
Commit: https://github.com/microsoft/vscode/commit/ef09cb02d48d9e995996ce63d1b3d68a9e03dce9

**Comment by Marcono1234** [comment, marked as answer]
Looks like this was added at some point at the top right of the commit view:
![Whitespace button screenshot](https://github.com/community/community/assets/11685886/1bc334a8-537e-4461-b1c4-0176b131e6f2)
